readme.md

```diff
@@ -0,0 +1,237 @@
+[path]
+workspace_base_dir = /<your_storage>/work_devel/build
+workspace_storage_base_dir = /<your_storage>/storage
+workspace_cache_base_dir = /<your_storage>/work_devel/tmp
+[local_conf]
+XT_GUESTS_INSTALL = "domu"
+XT_GUESTS_BUILD = "domu"
+```
+Pay attention that you need to specify your own drive with about 500GB of free space.
+
+Now you can run script:
+```
+python ./build_prod.py --machine h3ulcb-4x2g-kf --product devel --config local-devel-u.cfg --with-local-conf --with-do-build
+```
+
+### Few build hints
+Pay attention that mentioned above command will erase `workspace_base_dir` and start clear build.
+
+* If case of interruption you can just re-run script keeping all fetched sources:
+```
+python ./build_prod.py --machine h3ulcb-4x2g-kf --product devel --config local-devel-u.cfg --continue-build
+```
+
+* You can get "essence" of prod-aos, apply some pathes/modifications and continue build:
+```
+python ./build_prod.py --machine h3ulcb-4x2g-kf --product devel --config local-devel-u.cfg --with-local-conf
+pushd <workspace_base_dir>/meta-xt-prod-devel
+# do something, apply patches
+popd
+python ./build_prod.py --machine h3ulcb-4x2g-kf --product devel --config local-devel-u.cfg --continue-build
+```
+
+* To clear build products and intermediate files:
+```
+rm -rf <workspace_cache_base_dir>
+cd <workspace_base_dir>/build
+# remove everything except conf/ folder
+rm -rf bitbake.lock buildhistory/ cache/ deploy/ log/ shared_rootfs/ tmp/
+```
+
+## How to install
+
+### Write to microSD card
+As already mentioned in Quick start section, you can use `dd` or `bmaptool`.
+```
+sudo dd if=./image.img of="/dev/sd?" bs=1M status=progress
+```
+or bmaptool which is strongly recommended due to higher speed
+```
+sudo bmaptool copy --bmap image.img.bmap ./imgage.img /dev/sd?
+```
+
+### Write to eMMC
+We have two ways to put binaries onto eMMC: using SD card or using TFTP and NFS.
+
+#### Write to eMMC using SD card
+Detailed instruction with manual and scripted option is provided on wiki page
+https://github.com/xen-troops/meta-xt-prod-devel/wiki/How-to-flash-image-to-eMMC-from-SD-card
+
+#### Write to eMMC using network
+We need to assume that you have TFTP and NFS servers working on your PC.
+TODO Add page to wiki and put link here
+
+## Supported boards
+List of supported MACHINE_NAMEs:
+```
+MACHINE_NAME         Board name         SoC       RAM  prod-devel-unsafe only?  DomA supported?
+------------------------------------------------------------------------------------------------------
+salvator-x-h3-4x2g   Salvator-X         H3 ES3.0  8GB
+salvator-x-m3        Salvator-X         M3        4GB  prod-devel-unsafe only   Not supported for DomA
+salvator-xs-h3-4x2g  Salvator-XS        H3 ES3.0  8GB
+salvator-xs-m3n      Salvator-XS        M3N       2GB                           Not supported for DomA
+h3ulcb-4x2g          H3ULCB             H3 ES3.0  8GB
+h3ulcb-4x2g-kf       H3ULCB KingFisher  H3 ES3.0  8GB
```

Comment:
When re-basing this on new master (REL-v7.0) please don't forget to add salvator-xs-m3-2x4g

Reply:
I added mentioned board with comment that it is supported from REL 7.0